src/app/technical-record/notes/notes.component.html

```diff
@@ -1,22 +1,31 @@
 <dl class="govuk-summary-list">
-  <div class="govuk-summary-list__row" *ngIf="activeRecord.vehicleType === 'hgv' || activeRecord.vehicleType === 'trl'">
+  <div class="govuk-summary-list__row" *ngIf="activeRecord.vehicleType !== 'psv'">
     <dt class="govuk-summary-list__key">Notes</dt>
     <dd id="test-notes" class="govuk-summary-list__value">
-      <span id="test-notes" class="notes-section">{{ activeRecord.notes | DefaultNullOrEmpty }}</span>
+      <span id="test-notes" class="notes-section">{{
+        activeRecord.notes | DefaultNullOrEmpty
+      }}</span>
     </dd>
   </div>
 
   <div class="govuk-summary-list__row" *ngIf="activeRecord.vehicleType === 'psv'">
     <dt class="govuk-summary-list__key">Remarks</dt>
     <dd id="test-dispensations" class="govuk-summary-list__value">
-      <span id="test-notes" class="notes-section">{{ activeRecord.remarks | DefaultNullOrEmpty }}</span>
+      <span id="test-notes" class="notes-section">{{
+        activeRecord.remarks | DefaultNullOrEmpty
+      }}</span>
     </dd>
   </div>
 
-  <div class="govuk-summary-list__row" *ngIf="activeRecord.vehicleType === 'psv'">
+  <div
+    class="govuk-summary-list__row"
+    *ngIf="activeRecord.vehicleType !== 'trl' && activeRecord.vehicleType !== 'hgv'"
```

Comment:
if we were to move the conditional at the parent level this won't be needed anymore